pytest_selenium/drivers/remote.py

```diff
@@ -9,7 +9,9 @@
 
 
 def driver_kwargs(capabilities, firefox_profile, host, port, **kwargs):
```

Comment:
Should we also fix this for the appium driver: https://github.com/pytest-dev/pytest-selenium/blob/a91b1f5e0252bb10654516f3bb73598b11a72e22/pytest_selenium/drivers/appium.py#L12 ?

Reply:
Yeah, I forgot about appium 😅 I'll do that